lib/pf/radius.pm

```diff
@@ -122,6 +122,12 @@ sub authorize {
     # There is activity from that mac, call node wakeup
     node_mac_wakeup($mac);
 
+    # Handling machine auth detection
+    if ( defined($user_name) && $user_name =~ /host\// ) {
```

Comment:
A sAMAccountName cannot contain a forward slash.

http://social.technet.microsoft.com/wiki/contents/articles/11216.active-directory-requirements-for-creating-objects.aspx#Objects_with_sAMAccountName_Attribute